internal/subshell/cmd/env.go

```diff
@@ -34,47 +34,6 @@ func getEnvironmentPath(userScope bool) string {
 	return `SYSTEM\ControlSet001\Control\Session Manager\Environment`
 }
 
-// unsetUserEnv clears a state cool configured environment variable
-// It only does this if the value equals the expected value (meaning if we can verify that state tool was in fact
-// responsible for setting it)
-func (c *CmdEnv) unset(name, ifValueEquals string) *failures.Failure {
```

Comment:
Wouldn't this still be relevant? Restoring a backup is only part of the logic here.

Reply:
Good point.  I mis-understood that function initially.